src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -273,7 +273,7 @@ export const ImportBlueprintModal: React.FunctionComponent<
             <HelperText>
               <HelperTextItem
                 variant={
-                  isRejected ? 'error' : isOnPrem ? 'warning' : 'default'
+                  isRejected ? 'error' : isInvalidFormat ? 'error' : isOnPrem ? 'warning' : 'default'
```

Comment:
I'll just add that if you decide to go with the ternary operator, we could remove one level of nesting by chaining the `isRejected` and `isInvalidFormat` since they both lead to `'error'` variant.

```suggestion
                  isRejected || isInvalidFormat ? 'error' : isOnPrem ? 'warning' : 'default'
```